cosmpy/aerial/client/staking.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2021 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+from enum import Enum
+
+from cosmpy.crypto.address import Address
+from cosmpy.protos.cosmos.base.v1beta1.coin_pb2 import Coin
+from cosmpy.protos.cosmos.staking.v1beta1.tx_pb2 import (
+    MsgBeginRedelegate,
+    MsgDelegate,
+    MsgUndelegate,
+)
+
+
+class ValidatorStatus(Enum):
+    UNSPECIFIED = "BOND_STATUS_UNSPECIFIED"
+    BONDED = "BOND_STATUS_BONDED"
+    UNBONDING = "BOND_STATUS_UNBONDING"
+    UNBONDED = "BOND_STATUS_UNBONDED"
+
+    @classmethod
+    def from_proto(cls, value: int) -> "ValidatorStatus":
```

Comment:
why quoting type here? 
My first guess is to prevent the import, but this is still weird, it's quite useful to be able to look up imports when trying to answer "where is XYZ used?"

Reply:
So IDEs should work it out this is a python thing. Unquoting it would actually lead to a circular dependency thing. I have never truly understood the parsing logic of the python interpreter - I just have a quote it when it generates an error 😁

Reply:
Ok, that's one I'll dig up later then :)